### DISCUSSION TITLE
how should i send string Array

### DISCUSSION THREAD

**Discussion by Ali-N97** [jchristn/SuperSimpleTcp#168] [Q&A]
Hi i want to send  items of listbox from client to listbox of server and reverse. how should i do that.

**Comment by Ali-N97** [comment]
i used String.join and Split and it worked

**Reply by jchristn** [reply]
Nice!  There are a variety of ways you can do it, it really comes down to being able to control both the encode and the decode.  You may also consider serializing to and deserializing from either JSON or XML.  Cheers.
